lib/scraper.js

```diff
@@ -107,7 +115,11 @@ async function description(description_URL) {
         'The Office of Graduate Academic Affairs is responsible for the implementation and enforcement of graduate academic policies, the completion of academic standing reviews, degree audits and degree conferrals, communication with and counseling of graduate students, and the identification of campus resources, as appropriate to graduate student needs.'
       );
     } else if (description_URL[i] === 'https://web.iit.edu/wiser') {
-      descriptions.push(document.querySelector('.col-xs-12 p').textContent);
+      document.querySelectorAll('p').forEach((e) => {
+        if (e.textContent.includes('The Wanger Institute for Sustainable Energy Research (WISER) serves')) {
+          descriptions.push(e.textContent);
+        }
+      });
```

Comment:
Could we add description to the office object instead? Since the sizes of the two separated arrays don't match each other currently, we don't know which description belongs to which office. PS. For now there are 58 locations v.s. 26 descriptions.